The description parameter of @server should be optional (#4804)
microsoft/typespec: .chronus/changes/serverDescOption-2024-9-21-17-14-49.md

---
changeKind: fix
packages:
  - "@typespec/http"
---

> The description parameter of @server should be optional.

Reviewer: ```suggestion
The description parameter of `@server` is now optional
```
